lib/wiki_api.rb

```diff
@@ -77,6 +77,34 @@ def get_article_rating(titles)
     ArticleRatingExtractor.new(pages).ratings
   end
 
+  def deleted_logs(last_run_timestamp)
```

Comment:
I think this should live outside of WikiApi. It might be a good opportunity to refactor it so that doing a query that uses `continue` doesn't require calling a private method directly.

Reply:
So I should make `fetch_all` public and move these methods to a different(new?) file?